wellcomeml/ml/clustering.py

```diff
@@ -15,7 +15,7 @@
 try:
     from hdbscan import HDBSCAN
     HDBSCAN_INSTALLED = True
-except ValueError:
+except ModuleNotFoundError:
```

Comment:
I think we need to catch both errors actually.

If hdbscan is installed (but not installed with the right flags), it will still throw a value error as per https://github.com/wellcometrust/WellcomeML/issues/197. 

So maybe `except (ValueError, ModuleNotFoundError)`.

Reply:
394062a